src/App.js

```diff
@@ -1,24 +1,74 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
+import React from "react";
+import "./App.css";
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import SignupContainer from "./components/SingUp/SingUpFormContainer";
+import LoginContainer from "./components/Login/LoginFormContainer";
+import UploadImages from "./components/ImageUpload/ImageUploadContainer";
+import CreateStudioContainer from "./components/Studio/CreateStudioContainer";
+import StudioListContainer from "./components/Studio/StudioListContainer";
+import StudioDetails from "./components/Studio/StudioDetails";
+import MyStudioContainer from "./components/Studio/MyStudioContainer";
+import UpdateStudioContainer from "./components/Studio/UpdateStudioContainer";
+import CreateProjectContainer from "./components/Project/CreateProjectContainer";
+import CreateWorkshopContainer from "./components/Workshop/CreateWorkshopContainer";
+import WorkshopListContainer from "./components/Workshop/WorkshopListContainer";
+import Toolbar from "./components/Toolbar/Toolbar";
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <Toolbar></Toolbar>
+        <Link to="/signup">
+          <li>~ SIGN UP ~ </li>
+        </Link>
+        <Link to="/login">
+          <li>~ LOGIN ~</li>
+        </Link>
+        <Link to="/uploadImage">
+          <li>~ UPLOAD IMAGE ~</li>
```

Comment:
I'm not sure these need to be `<li>` items, have you tried just with text in the Link?

Reply:
Also, all these links you can extract to a NavBar component for example and render that component here so your App.js is cleaner